Assuming unrestricted shared filesystem usage.
None
host: Allens MacBook
Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Job stats:
job                      count
---------------------  -------
all                          1
download_annotation          1
download_quantitation        1
make_intervals               1
write_lr_records             1
total                        5

Select jobs to execute...
Execute 1 jobs...
[Thu Sep 18 17:50:36 2025]
localrule make_intervals:
    input: isoformDB/references/grch38.gencode.v29/gene_annotation.gtf.gz
    output: isoformDB/references/grch38.gencode.v29/cannonical_genes.gff
    jobid: 4
    reason: Missing output files: isoformDB/references/grch38.gencode.v29/cannonical_genes.gff; Updated input files: isoformDB/references/grch38.gencode.v29/gene_annotation.gtf.gz
    wildcards: reference_id=grch38.gencode.v29
    resources: tmpdir=/var/folders/tk/bdsgytmn5cb0bfqrftv10n8r0000gq/T, mem_mb=2000, mem_mib=1908, time=00:30:00, cpus_per_task=2
Shell command: 
        isoformer-select-canonical isoformDB/references/grch38.gencode.v29/gene_annotation.gtf.gz > isoformDB/references/grch38.gencode.v29/cannonical_genes.gff
        
Select jobs to execute...
RuleException:
CalledProcessError in file "/Users/allen/projects/dna_lang/pytorch_borzoi/isoformer/database/pipeline/main.smk", line 194:
Command 'set -euo pipefail;  
        isoformer-select-canonical isoformDB/references/grch38.gencode.v29/gene_annotation.gtf.gz > isoformDB/references/grch38.gencode.v29/cannonical_genes.gff' returned non-zero exit status 1.
[Thu Sep 18 17:50:39 2025]
Error in rule make_intervals:
    message: None
    jobid: 4
    input: isoformDB/references/grch38.gencode.v29/gene_annotation.gtf.gz
    output: isoformDB/references/grch38.gencode.v29/cannonical_genes.gff
    shell:
        
        isoformer-select-canonical isoformDB/references/grch38.gencode.v29/gene_annotation.gtf.gz > isoformDB/references/grch38.gencode.v29/cannonical_genes.gff
        
        (command exited with non-zero exit code)
Removing output files of failed job make_intervals since they might be corrupted:
isoformDB/references/grch38.gencode.v29/cannonical_genes.gff
Shutting down, this might take some time.
Exiting because a job execution failed. Look below for error messages
[Thu Sep 18 17:50:39 2025]
Error in rule make_intervals:
    message: None
    jobid: 4
    input: isoformDB/references/grch38.gencode.v29/gene_annotation.gtf.gz
    output: isoformDB/references/grch38.gencode.v29/cannonical_genes.gff
    shell:
        
        isoformer-select-canonical isoformDB/references/grch38.gencode.v29/gene_annotation.gtf.gz > isoformDB/references/grch38.gencode.v29/cannonical_genes.gff
        
        (command exited with non-zero exit code)
Complete log(s): /Users/allen/projects/dna_lang/pytorch_borzoi/isoformer/database/pipeline/.snakemake/log/2025-09-18T175036.280017.snakemake.log
WorkflowError:
At least one job did not complete successfully.
